Header header
string id
string cid
string type
string status
float32 battery
std_msgs/Time guiStamp
string guiTaskName
string guiTaskValue
geometry_msgs/PoseStamped guiGoalPoint
geometry_msgs/PoseStamped odometry
GoalCompressed goal
AgentReset reset
uint16 numDiffs
uint16 latestPoseGraph
BeaconArray commBeacons
std_msgs/Time baseStamp
AgentArtifact[] baseArtifacts
marble_artifact_detection_msgs/ArtifactArray newArtifacts
std_msgs/Time lastMessage
NeighborMsg[] neighbors
